src/main/java/bio/terra/app/controller/DatasetsApiController.java

```diff
@@ -214,6 +177,15 @@ private AuthenticatedUserRequest getAuthenticatedInfo() {
         return jobToResponse(jobService.retrieveJob(jobId, userReq));
     }
 
+    @Override
+    public ResponseEntity<JobModel> applyDatasetDataDeletion(
+        String id,
+        @RequestBody @Valid bio.terra.model.DataDeletionRequest dataDeletionRequest) {
```

Comment:
Is a fully qualified name needed here?
```suggestion
        @RequestBody @Valid DataDeletionRequest dataDeletionRequest) {
```

Reply:
Nope, looks like that got auto-generated